src/brucke_app.erl

```diff
@@ -1,5 +1,5 @@
 %%%
-%%%   Copyright (c) 2016 Klarna AB
+%%%   Copyright (c) 2017 Klarna AB
```

Comment:
2017 for newly added files.
2016-2017 for modified files.
Do not change a file only to update Copyright notice years.